tools/cli/go-whisk-cli/commands/util.go

```diff
@@ -68,33 +68,41 @@ Examples:
       /ns/foo => qName {namespace: ns, entityName: foo}
       /ns/pkg/foo => qName {namespace: ns, entityName: pkg/foo}
 */
-func parseQualifiedName(name string) (qName qualifiedName, err error) {
+func parseQualifiedName(name string) (QualifiedName) {
```

Comment:
No longer returning an error... Isn't it possible for an entity name to be incorrectly formatted?  For example:
  `wsk action get ""`
The deprecated python CLI returned `error: entity name missing, did you mean to list collection`